pkg/types/field.go

```diff
@@ -162,6 +162,12 @@ func NewField(g *Builder, cfg *config.Resource, r *resource, sch *schema.Schema,
 		}
 	}
 
+	for _, ignoreField := range cfg.LateInitializer.ConditionalIgnoredFields {
+		if ignoreField == traverser.FieldPath(f.TerraformPaths) {
+			cfg.LateInitializer.AddConditionalIgnoredCanonicalFields(traverser.FieldPath(f.CanonicalPaths))
+		}
+	}
+
```

Comment:
How about accepting the actual CRD canonical path during the configuration? Then we would not need to do the conversion here.